feat: footer

err53 opened the pull request:
Closes #10

~~Waiting on profile data from DeltaHacks members~~ will be added in a forthcoming PR

## Requirements
- [x] Dinos Image
- [x] Team Member Marquee
    - [x] Team Member Quotes
- [x] Social Links on Mobile
    - [x] Make the social links actually clickable
    - [x] Update social link icon based on revisions from Design team
- [x] Footer Links

err53 committed:
feat: add slider boilerplate

err53 committed:
Merge branch 'main' into footer

err53 committed:
Merge branch 'main' into footer

err53 committed:
feat: add first iteration of footer

err53 committed:
feat: refactoring, support responsive

Krish120003 commented on src/sections/Footer.tsx:
```diff
@@ -1,20 +1,39 @@
+        <Link
+          href="https://github.com/DeltaHacks/deltahacks-website-2024"
+          className="hover:underline"
+        >
```
This is a flexbox, this should be using gap for spacing instead of margins from space-x-8.

err53 replied:
I fixed it in the commit directly before your review :)

err53 committed:
chore: refactor slider -> team marquee

err53 committed:
fix: remove extraneous space

err53 committed:
feat: add team member quotes

err53 committed:
feat: add transitions for hover

err53 committed:
fix: properly pause marquee when hovering over quote

err53 committed:
feat: add footer social icons

err53 committed:
fix: remove unused import

err53 committed:
feat: add social links

note: We don't have a devpost for DH11 yet, so I'm linking to a list of the previous ones

err53 committed:
fix: allow marquee to span full width of screen

err53 committed:
fix: update social icons as per Design team

err53 committed:
Disable Marquee

Krish120003 committed:
fix: update links

Krish120003 committed:
Merge branch 'main' into footer

Krish120003 commented:
Opted to hide the Team section until we have the images from everyone

kabirsg commented:
> Opted to hide the Team section until we have the images from everyone

We should have everyone's picture by Sunday EOD. Ill send a reminder out at some point on Sunday too but hopefully we can get it by then

err53 committed:
fix: add pink bg and transition

This should be updated to be purple in the future

kabirsg commented:
This looks good to me. Btw the privacy policy and code of conduct are good and there's nothing else to add. No travel policy or anything like that.

Krish120003 commented on next.config.js:
```diff
@@ -30,6 +30,10 @@ const config = {
     ];
   },
+  images: {
+    dangerouslyAllowSVG: true,
```
Why do we need this?

Krish120003 committed:
fix: remove unnecessary config

Krish120003 committed:
Merge branch 'main' into footer

kabirsg reviewed (approved):
Looks good to me
